src/token.rs

```diff
@@ -65,7 +69,7 @@ pub async fn lookup(
             context
                 .sql
                 .query_get_value(
-                    "SELECT token FROM tokens WHERE namespc=? AND foreign_id=?;",
+                    "SELECT token FROM tokens WHERE namespc=? AND foreign_id=? ORDER BY timestamp DESC LIMIT 1;",
```

Comment:
i feel like we should describe somewhere when a duplicate would occur.  maybe just a comment in front of this query?